src/app/container/launch/launch.component.html

```diff
@@ -38,10 +38,14 @@
       <pre>{{ cli }}</pre>
     </div>
     <div *ngIf="(currentDescriptor | lowercase) === 'cwl'">
-      <div class="well well-sm" tooltip="Commands for launching tools through CWLtool: the CWL reference implementation.
-      cwltool can run a tool directly when all inputs and outputs are available on the local filesystem.">
-        <a href="https://github.com/common-workflow-language/cwltool">Run locally with cwltool</a>
+      <div class="well well-sm" tooltip="Commands for launching tools through cwl-runner: the CWL reference implementation.
```

Comment:
`cwl-runner` is the generic way to run a CWL description using any CWL aware system. The name of the CWL reference runner is `cwltool`

Reply:
so the second half of the sentence should read "the generic interface to many CWL implementations."